EIPS/eip-xxxx.md

```diff
@@ -0,0 +1,1118 @@
+---
+EIP: TBD
+
+Title: Royalty Bearing NFTs
+
+Description: This proposal outlines the structure of an extension of the ERC721 standard for NFTs to correctly define, process, and pay (hierarchical) onchain royalties from NFT sales, and goes beyond [EIP-2981](https://eips.ethereum.org/EIPS/eip-2981).
+
+Author: Andreas Freund (@Therecanbeonlyone1969) 
+
+discussions-to: https://ethereum-magicians.org/t/royalty-bearing-nfts/8453
+
+Status: Draft
+
+Type: Standards Track
+
+Category: ERC
+
+Created: 2022-03-14
+
+Requires: ERC721, ERC165, ERC2771
```

Comment:
No newlines between each of these.

Reply:
Also the field names should be all lower-case.

Reply:
done